sematic/api/endpoints/meta.py

```diff
@@ -43,8 +43,8 @@ def env_endpoint(user: Optional[User]) -> flask.Response:
     env = {}
     for settings in (
         ServerSettingsVar.GOOGLE_OAUTH_CLIENT_ID,
-        ServerSettingsVar.KUBERNETES_NAMESPACE,
```

Comment:
I don't think it actually was meant to return all settings, it returned a subset from its initial implementation. There are security implications of exposing all server configurations to clients by default, as server configurations can sometimes contain sensitive info. Rather than scrubbing sensitive values (where it's easy to miss one) a safer approach is to only expose ones you know deserve exposure. But as a broader point, this PR doesn't change anything about the "expose all or a subset" behavior, it just stays consistent with what's already there. If we think that should be changed, it should be done in a separate PR.